OrbitGl/TimeGraph.cpp

```diff
@@ -263,25 +259,26 @@ void TimeGraph::ProcessTimer(const TimerInfo& timer_info, const FunctionInfo* fu
 
   if (timer_info.type() == TimerInfo::kGpuActivity) {
     uint64_t timeline_hash = timer_info.timeline_hash();
-    std::shared_ptr<GpuTrack> track = GetOrCreateGpuTrack(timeline_hash);
+    auto track = track_manager_->GetOrCreateGpuTrack(timeline_hash);
```

Comment:
I thought about that and finally used auto, my bad. I only used auto because it's easier to switch from different kinds of pointers, but you are right.